C#/Symbol List Classes.tmPreferences

```diff
@@ -4,11 +4,21 @@
 	<key>name</key>
 	<string>Symbol List: Classes</string>
 	<key>scope</key>
-	<string>source.cs meta.class.identifier</string>
+	<string>source.cs entity.name.class</string>
 	<key>settings</key>
 	<dict>
 		<key>showInSymbolList</key>
 		<integer>1</integer>
+		<key>showInIndexedSymbolList</key>
+		<string>1</string>
+		<key>symbolTransformation</key>
+		<string>
+		s/^/class /;
+		</string>
+		<key>symbolIndexTransformation</key>
+		<string>
+		s/^/class /;
```

Comment:
This schema (`symbolIndexTransformation` adding something to the indexed symbol) will prevent Go To Definition from functioning if applied on a class/method/enum name. It's fine for the local symbol list opened with ctrl+r, but the project symbol list and go to definition both use the indexed symbols, as far as I'm aware.